src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -2382,9 +2382,6 @@ internal Exception BindRunspace(Runspace initializedRunspace, PSTraceSource runs
             // Setting the module to null fixes that.
             initializedRunspace.ExecutionContext.EngineSessionState.Module = null;
 
-            // Set the SessionStateDrive here since we have all the provider information at this point
-            SetSessionStateDrive(initializedRunspace.ExecutionContext, true);
-
```

Comment:
How the changes relate to `Microsoft.ApplicationInsights`?

Reply:
This is a minor optimization in CPU time by removing redundant code. `BindRunspace` is called only from `DoOpenHelper()`, after `new AutomationEngine(Host, InitialSessionState)`. `SetSessionStateDrive(context, true)` has already been called in the constructor of `AutomationEngine`.